lmdeploy/turbomind/turbomind.py

```diff
@@ -350,6 +360,14 @@ def from_pretrained(cls,
                 Can be used to update configuration when initialize the engine.
         """
         model_source = get_model_source(pretrained_model_name_or_path)
+        # try fuzzy matching to get a model_name
+        if model_name is None and model_source == ModelSource.HF_MODEL:
+            potential_names = best_match_model(pretrained_model_name_or_path)
+            if potential_names is None:
```

Comment:
maybe we should assert potential_names

Reply:
It should make sense that huggingface weights, such as base models, can be converted to turbomind without `model_name`.
Besides, the current codes will assert it in the afterward conversion.